src/models/resource.ts

```diff
@@ -1,8 +1,25 @@
 import { ConnectionType } from "../clients/sidecar";
-import { IconNames } from "../constants";
+import { CCLOUD_CONNECTION_ID, IconNames, LOCAL_CONNECTION_ID } from "../constants";
 
 /** A uniquely-branded string-type for a connection ID. */
 export type ConnectionId = string & { readonly brand: unique symbol };
+
+/** Likewise for environment ids. Note that Direct Connection ids also double as environment ids. */
```

Comment:
This may get a little weird when swapping between the two branded types for `DIRECT` connections 😅 
But glad we have this now!